azure/become-deployment-user.yml

```diff
@@ -9,24 +9,22 @@ parameters:
 
 steps:
 
-  - task: DownloadSecureFile@1
-    name: aws_config
-    inputs:
-      secureFile: ptl_aws_config
-
   - bash: |
-      mkdir -p ~/.aws
-      mv $(aws_config.secureFilePath) ~/.aws/config
-      chmod 400 ~/.aws/config
-      ls -al ~/.aws/config
-    displayName: "Get AWS config"
+      set -e
```

Comment:
might be worth setting a `pipefail` here too in case anything untoward happens in any of the `$(...)` or redirections

Reply:
done